### PR TITLE
Fix height table header

### LINKED ISSUE #114: [Filter] Add max-heigth for each field in header of datatable

### PR DESCRIPTION
**Pull request by Alketta**
Close #114

### CODE REVIEW

File: front/src/components/store/ViewButton/ViewButton.stories.js [hunk 1]
```diff
@@ -5,57 +5,49 @@ export default {
...
+export const OverlayOpenedStoryWithViews = TemplateWithRef.bind({})
+OverlayOpenedStoryWithViews.storyName = 'overlay opened with views'
+OverlayOpenedStoryWithViews.args = {
+  views: createViews(3),
+  value: '1',
+}
 
-overlayOpenedStoryWithViews.storyName = 'overlay opened with views'
-overlayOpenedStoryWithViews.args = {
-  waitForSelector: '.p-overlaypanel',
```
Comment (mdartic) [inline front/src/components/store/ViewButton/ViewButton.stories.js:60]: I see you remove the `waitForSelector`, but imageshot seems "light",
Do you try with this special argument too, and were the imageshots the same ?
If so, that's ok for me to remove the `waitForSelector`.

File: front/src/components/store/ViewColumnButton/ViewColumnButton.stories.js [hunk 2]
```diff
@@ -6,75 +6,59 @@ export default {
...
+export const WithValueAndColumnsStory = TemplateWithRef.bind({})
+WithValueAndColumnsStory.storyName = 'with value and columns'
+WithValueAndColumnsStory.args = {
+  value,
+  columns: createOptions(5),
+}
 
-withValueAndColumnsStory.storyName = 'with value and columns'
-withValueAndColumnsStory.args = {
-  waitForSelector: '.p-overlaypanel',
```
Comment (mdartic) [inline front/src/components/store/ViewColumnButton/ViewColumnButton.stories.js:79]: Same here

**Review by mdartic — COMMENTED** [review]
Except the `waitForSelector`, seems good to me!

File: front/src/components/store/ViewButton/ViewButton.stories.js (hunk 1 above)
Comment (Alketta) [reply front/src/components/store/ViewButton/ViewButton.stories.js:60]: My bad, I didn't know it was useful

Comment (mdartic) [reply front/src/components/store/ViewButton/ViewButton.stories.js:60]: Well, in fact it's a helper to tell puppeteer wait for this special element before taking screenshot.
I don't know if it "works" really, but that help us to always have the "better" imageshot before sending to CI.

File: front/src/components/store/FilterButton/FilterButton.stories.js [hunk 3]
```diff
@@ -124,45 +122,57 @@ const columnsDropdownOptions = {
...
+const TemplateWithManyFiltersOpened = (args, { argTypes }) => ({
+  components: { FilterButton },
+  props: Object.keys(argTypes),
+  template: '<FilterButton ref="fb" v-bind="$props" />',
+  async mounted () {
+    // Open the panel
+    this.$refs.fb.$el.querySelector('button').click()
+    // Delay
+    await Vue.nextTick()
+    // Add a new filter
```
Comment (tcitworld) [inline front/src/components/store/FilterButton/FilterButton.stories.js:159]:
```suggestion
    // Add several new filters
```

**Pull request merged**